### PR TITLE
Remove DisCo from transpilation to elkjs because it somehow breaks the CI build

### PR DESCRIPTION
**Pull request by Eddykasp**

### CODE REVIEW

**Comment by Eddykasp** [comment]
this confirms that the problem appears to limited to disco and its CompactionStrategy

**Comment by soerendomroes** [comment]
Does this mean we remove disco? Or that disco just cannot be transpiled to JavaScript?

**Comment by Eddykasp** [comment]
Removing it would fix the CI build on Github but since I can still build it locally, I don't think we should jump straight to removing it completely. But it's an option to keep in mind I guess

File: build.gradle [hunk 1]
```diff
@@ -71,7 +71,7 @@ def elkSources = [
   "${elkRepo}/plugins/org.eclipse.elk.graph.json/src",
   "${elkRepo}/plugins/org.eclipse.elk.alg.common/src",
   // The algorithms
-  "${elkRepo}/plugins/org.eclipse.elk.alg.disco/src",
+  //"${elkRepo}/plugins/org.eclipse.elk.alg.disco/src",
```
Comment (soerendomroes) [inline build.gradle:74]: Maybe add a comment why this is not included here and maybe a FIXME to investifate the problem.

File: src/java/org/eclipse/elk/js/ElkJs.java [hunk 2]
```diff
@@ -24,7 +24,7 @@
 import org.eclipse.elk.graph.json.*;
 
 import org.eclipse.elk.alg.common.compaction.options.*;
-import org.eclipse.elk.alg.disco.options.*;
+//import org.eclipse.elk.alg.disco.options.*;
```
Comment (soerendomroes) [inline src/java/org/eclipse/elk/js/ElkJs.java:27]: See above

File: src/java/org/eclipse/elk/js/ElkJs.java [hunk 3]
```diff
@@ -144,8 +144,8 @@ public static void registerLayoutAlgorithms(final JavaScriptObject arrayObj) {
                 SERVICE.registerLayoutMetaDataProviders(new MrTreeMetaDataProvider());
             } else if (alg.equals("radial")) {
                 SERVICE.registerLayoutMetaDataProviders(new RadialMetaDataProvider());
-            } else if (alg.equals("disco")) {
-                SERVICE.registerLayoutMetaDataProviders(new PolyominoOptions(), new DisCoMetaDataProvider());
+   //         } else if (alg.equals("disco")) {
```
Comment (soerendomroes) [inline src/java/org/eclipse/elk/js/ElkJs.java:147]: See above

**Review by soerendomroes — CHANGES_REQUESTED** [review]

**Review by soerendomroes — APPROVED** [review]

**Pull request merged**